File: C/逆序输出字符串.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
int main(int argc,char** argv)
{
	FILE *fp;
	if((fp=fopen("result.txt","w"))==NULL)
		return -1;
	char a[11];
	int i,t;
	t=strlen(argv[1]);
	if(t>10) t=10;
	for(i=1;i<=t;i++)
	{
		a[i]=argv[1][t-i];
		fprintf(fp,"%c",a[i]);
	}
	fclose(fp);	
	return 0;
}

File: Python/逆序输出字符串.py
import sys
t,a=len(sys.argv[1]),[]
if t>10 :
	t=10
for i in range(1,t+1):
	a.append(sys.argv[1][t-i])
with open('result.txt','w') as fp:
	for i in range(t):
		fp.write(a[i])